lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -324,6 +324,12 @@ mixin _WheelEventListenerMixin on _BaseAdapter {
     const int domDeltaLine = 0x01;
     const int domDeltaPage = 0x02;
 
+    // Logic from https://stackoverflow.com/a/62415754
+    ui.PointerDeviceKind kind = ui.PointerDeviceKind.mouse;
+    if (event.deltaX % 120 != 0 || event.deltaY % 120 != 0) {
```

Comment:
Where did you get the 120 value?  Maybe add a comment explaining that (like what device and browser, if that's what you did).

Reply:
It's just a convention which dates from Windows I think, not really documented but it holds on all major browsers. I'll update the comment.

Reply:
Neat, I never knew that 👍